tests/estimators/online/test_dual_ukf.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+
+from moirae.models.ecm import EquivalentCircuitModel as ECM
+from moirae.models.ecm.advancedSOH import ECMASOH
+from moirae.models.ecm.ins_outs import ECMInput
+from moirae.models.ecm.transient import ECMTransientVector
+from moirae.simulator import Simulator
+from moirae.estimators.online.dual import DualEstimator
+
+
+def cycle(start_time=0):
+    # Charge for 3 hours at a current of 3 A
+    chg_time = 3 * 3600
+    final_time = start_time + chg_time
+    inputs = [ECMInput(time=time, current=3) for time in np.arange(start_time, final_time)]
+    # Now, discharge for 4 hours at a current of 2.25 A
+    dischg_time = 4 * 3600
+    final_time += dischg_time
+    inputs += [ECMInput(time=time, current=-2.25) for time in np.arange(start_time + chg_time, final_time)]
+    return final_time, inputs
+
+
+def test_names(simple_rint):
+    rint_asoh, rint_transient, ecm_inputs, ecm_model = simple_rint
+    rint_asoh.mark_updatable('r0.base_values')
+    ukf_dual = DualEstimator.initialize_unscented_kalman_filter(
+        cell_model=ecm_model,
+        initial_asoh=rint_asoh,
+        initial_transients=rint_transient,
+        initial_inputs=ecm_inputs,
+        covariance_transient=np.eye(2),
+        covariance_asoh=np.atleast_2d(1)
+    )
+    assert ukf_dual.state_names == ('soc', 'hyst', 'r0.base_values')
+    assert ukf_dual.output_names == ('terminal_voltage',)
+    assert ukf_dual.control_names == ('time', 'current')
+
+
+def test_simplest_rint():
+    # Create simplest possible Rint model, with no hysteresis, and only R0
```

Comment:
It was my intent to have "Simple_Rint" lack hysteresis. Did I accidentally have it in there still?

Reply:
No, you don't. I am not using the simple_rint fixture here because I initialize some of the values to something other than the default ones. I figured that was easier than pulling simple_rint, making copies (for real and guess objects) and changing the values. Would you prefer if I used the simple_rint?

Reply:
Nah. Just wanted to make sure I didn't make the "simple int" too complicated. If it doesn't work here, I don't oppose using something else